engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -493,6 +514,20 @@ public void setErrorMessage(String errorMessage)
       this.errorMessage = errorMessage;
     }
 
+    public String getLogFileName()
```

Comment:
I don't think other events will have not null logFileInfo, Let's not make it generic unless we are sure about it.